external-deps/spyder-remote-services/spyder_remote_services/services/files/compression.py

```
@@ -0,0 +1,998 @@
+
+        self._raise_if_beyond(
+            compressed_size,
+            maximum=max_compressed_size,
+            exception_class=CompressedSizeOverflowError,
+        )
+
+        yield from self.write_chunk(compressed_chunk)
+
+        return uncompressed_size, compressed_size, crc_32
+
+    def _zip_64_local_header_and_data(
```

Pylint (3.3.7)

[refactor] Too many arguments (10/5)

(R0913)